.circleci/config.yml

```diff
@@ -191,35 +191,40 @@ jobs:
     steps:
       - run: echo "hello from branch ${CIRCLE_BRANCH}"
 workflows:
+  ### on bors action or PR ###
```

Comment:
I thought we were just sticking to bors for now?

Reply:
I'd be happy with either route, but I think it is far too easy to forget about canary and I've seen folks starting running really long tests on their local dev box and missing things.

Reply:
I think we should have something lightweight at least for PR. Maybe just lint & unit test. Either way, these jobs won't actually run unless we set back the setting in CircleCI, so we can just land this for now IMO